index.html

```diff
@@ -35,5 +35,118 @@ <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
+        </article>
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button class="project-button" type="button" name="button">See project</button>
+        </article>
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button class="project-button" type="button" name="button">See project</button>
+        </article>
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button class="project-button" type="button" name="button">See project</button>
+        </article>
+      </div>
+    </main>
+
+    <section id="about" class="font-style">
+      <h3>About me</h3>
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hesitate to contact me.
+      </p>
+      <button class="project-button" type="button" name="button">Get my resume</button>
+      <div class="container">
+        <div class="grid-item">
+            <img src="https://fakeimg.pl/200x200" alt="">
+            <h4>Languages</h4>
+            <ul class="list-format container">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+        </div>
+        <div class="grid-item">
+            <img src="https://fakeimg.pl/200x200" alt="">
+            <h4>Frameworks</h4>
+            <ul class="list-format container">
+              <li>React.js</li>
+              <li>RoR</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+            </ul>
+        </div>
+        <div class="grid-item">
+            <img src="https://fakeimg.pl/200x200" alt="">
+            <h4>Skills</h4>
+            <ul class="list-format container">
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>Web Development</li>
+              <li>API Design</li>
+            </ul>
+        </div>
+    </section>
```

Comment:
- [ ] I see The following issues with this section. Kindly fix them
    - The Title - `About Me` currently has a font size of about 20px instead of 40px according to the Figma template. 
    - The paragraph (e.g., "I'm a software ...") currently has a font size of 16px, instead of 20px according to the Figma template. 
    - The button `Get my resume` should have a font size of 17px. 
    - There should be a background illustration for this section but you have not included it.